android/player/src/main/res/values/tags.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="view_hydrated" type="bool" />
```

Comment:
was this for the automation tests?

Reply:
Yeah, it's the only way we currently have to determine if an asset has been completely hydrated and ready for use